style.css

```diff
@@ -490,4 +523,47 @@ textarea::placeholder {
     align-items: flex-start;
     text-align: start;
   }
+
+  .section-4 {
+    background-color: #6070ff;
+    background-image: url("images/contact\ form\ background\ shapes@2x.png");
+    background-size: 90%;
+    background-position: center;
+    background-repeat: no-repeat;
+    border-top-left-radius: 100px;
+    width: 100%;
+    height: 100%;
+  }
```

Comment:
- [ ] As stated above about the responsiveness at all the desktop screen sizes. Please ensure that the Contact section is also responsive and its contents are user friendly and professional according to the Figma design ☺️.
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/236177854-f57458e4-37e9-4b5a-b932-77a505a29650.png)